crates/ethereum/node/src/lib.rs

```diff
@@ -21,7 +21,7 @@ pub use evm::{EthEvmConfig, EthExecutorProvider};
 
 pub use reth_ethereum_consensus as consensus;
 pub mod node;
```

Comment:
Maybe too much of a BC break, but since its re-exported, this should be private. Otherwise, there are two paths and its pretty confusing when trying to import something.

```suggestion
mod node;
```

Reply:
this would break existing imports that is also used currently, so we can keep this pub for now